crawl-ref/source/dat/dlua/layout/hyper_rooms.lua

```diff
@@ -111,7 +111,7 @@ function hyper.rooms.make_room(options,generator)
     profiler.push("AnalyseRoomInternal")
   end
 
-  if generator.analyse_interal then
+  if generator.analyse_internal then
```

Comment:
I think this is a bug in e58e97905fd38a6dc38414f42066f020df1e1665, or possibly leftover debugging code. It's never referenced anywhere, and never has been. (`analyse_internal` also doesn't exist.)